docs/quick_start.md

```diff
@@ -45,35 +46,40 @@ libtorch model which calculates the sum and difference of two inputs. Run the
 Model Analyzer `profile` subcommand inside the container with:
 
 ```
-$ model-analyzer profile -m /quick_start_repository/ \
-    --profile-models add_sub \
-    --run-config-search-max-concurrency 2 \
-    --run-config-search-max-instance-count 2 \
-    --run-config-search-preferred-batch-size-disable true
+$ model-analyzer profile -m . --profile-models add_sub
```

Comment:
Why do we need `-m .`, that is even worse. It obfuscates what it is asking. Should use the full option as well if you are using it.

Reply:
`model-analyzer profile` requires the user to supply the model option (`-m`):

```
$ model-analyzer profile --profile-models add_sub
2021-11-11 21:56:29.213 ERROR[entrypoint.py:214] Model Analyzer encountered an error: Config for model_repository is not specified. You need to specify it using the YAML config file or using the -m, --model-repository flags in CLI.
```

Reply:
reverted back to using `-m /quick_start_repository` 👍